README.md

```diff
@@ -46,7 +46,7 @@ Python bindings.
 
 ###  1. Introduction
 
-The mlpack website can be found at http://www.mlpack.org and contains numerous
+The mlpack website can be found at http://www.mlpack.org and it contains numerous
```

Comment:
I wouldn't say this is a grammatical error.

Reply:
Lol, you are right @ShikharJ. Do you mind telling me which projects need help in mlpack?

Reply:
I agree, technically it's ok as-is, but I'm also happy to merge both fixes.  The second one is definitely an issue. :)

Reply:
@rcurtin could you please tell me some projects that I can work on, other than the bugs? :)